### ISSUE TITLE
why we build another SimpleMLP network ?

### ISSUE THREAD

**Issue by tariqfahad** [datafrick/tutorial#2]
#to get the average over all the local model, we simply take the sum of the scaled weights
    average_weights = sum_scaled_weights(scaled_local_weight_list)
    
    #update global model 
    global_model.set_weights(average_weights)

    #test global model and print out metrics after each communications round
    for(X_test, Y_test) in test_batched:
        global_acc, global_loss = test_model(X_test, Y_test, global_model, comm_round)
        SGD_dataset = tf.data.Dataset.from_tensor_slices((X_train, y_train)).shuffle(len(y_train)).batch(320)
smlp_SGD = SimpleMLP()
SGD_model = smlp_SGD.build(784, 10) # why ?????

**Comment by SamiurRahman1** [comment]
Hi, as far as i understand, this is the global model. We need to initialize a global model and after that we update our global model by averaging the local models from clients.
